test/test_stablehlo_dump.py

```diff
@@ -14,7 +14,6 @@ def test_simple(self):
     z = x + y
     # Example usage of dumping StableHLO given output tensors
     stablehlo = xm.get_stablehlo([z])
-    self.assertEqual(stablehlo.count("stablehlo.multiply"), 1)
```

Comment:
why is this removed?

Reply:
Typo, the graph shouldn't have multiply op.